integration/tests/economy/test_economics.py

```diff
@@ -955,7 +955,7 @@ def test_deploy_contract_to_exist_unpayed(self):
             self.web3_client.send_neon(acc2, contract_address, 1, gas=1)
 
         _, contract_deploy_tx = self.web3_client.deploy_and_get_contract(
-            "Counter", "0.8.10", account=acc2
+            "common/Counter", "0.8.10", account=acc2
```

Comment:
There are two more `"Counter" -> "common/Counter"` and one more `"IncreaseStorage" -> "common/IncreaseStorage"` replacements needed.